Bluetooth: Fix EnOcean module parsing sensor data as commissioning (#14750)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

Bluetooth libraries and services
--------------------------------

* :ref:`bt_enocean_readme`:

>   * Fixed an issue where sensor data of a certain length would incorrectly be parsed as switch commissioning.

Reviewer: ```suggestion
  * Fixed an issue where the sensor data of a certain length was incorrectly parsed as switch commissioning.
```

Reviewer: Btw, do we need to be more specific about at what length(s) this issue would be observed, instead of saying 'certain length'?

Author: I'm not sure, but I can add it.
```suggestion
  * Fixed an issue where the sensor data of length 18 bytes was incorrectly parsed as switch commissioning.
```
?

Reviewer: of 18 bytes length :)
